share/models/ingest.py

```diff
@@ -372,10 +380,10 @@ class RawDatum(models.Model):
     # The sha256 of the datum
     sha256 = models.TextField(validators=[validators.MaxLengthValidator(64)])
 
-    datestamp = models.DateTimeField(null=True, help_text=(
+    datestamp = models.DateTimeField(default=pendulum.now, help_text=(
```

Comment:
`auto_now*` will override values set before saving, which is bad. The default here is just a fallback if it somehow got this far without a sensible datestamp.

`pendulum.now` defaults to UTC, yes, but does that matter? 08:00-05 is the same as 13:00+00 is the same as 17:00+04